src/main/resources/application.properties

```diff
@@ -34,7 +34,7 @@ info.app.version=${version:0.0.1}
 #
 # HMRC endpoints
 #
-base.hmrc.url=https://test-developer.service.hmrc.gov.uk
+base.hmrc.url=http://localhost:4000
```

Comment:
Have you checked this in by accident when testing locally with the stub?

Reply:
Your absolutely correct, I have.